src/test/java/com/artipie/vertx/VertxSliceServerTest.java

```diff
@@ -79,14 +77,11 @@ public void basicGetRequest() throws IOException {
         final String expected = "Hello World!!!";
         try (VertxSliceServer server = new VertxSliceServer(
             vertx,
-            (line, headers, body) -> connection -> {
-                final int okay = 200;
-                connection.accept(
-                    okay,
-                    Collections.emptyList(),
-                    Flowable.fromArray(ByteBuffer.wrap(expected.getBytes()))
-                );
-            },
+            (line, headers, body) -> connection -> connection.accept(
```

Comment:
@olegmoz Isn't there a way to test the promise that results from the Connection (or the Response.send)?

Reply:
@HDouss I don't think there is a way and that is really needed. VertxSliceServerTest tests VertxSliceServer and it does not expose future produced by Slice. Here we are testing that server   actually sends response that Slice sent to connection. We also need to handle errors in Slice, but I will add it later in other PR. This PR is just to adapt VertxSliceServer  to updated HTTP module.